### ISSUE TITLE
Port ckeygen to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8855]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8855|
|Type|enhancement|
|Created|2016-10-08 19:46:15Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8855 8855
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1475955975455174 1475955975455174
changetime__1500829224536505 1500829224536505
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/557

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset fdf427045e51ec2e2b2944521b0dae15a351e6aa
```
#!CommitTicketReference repository="" revision="fdf427045e51ec2e2b2944521b0dae15a351e6aa"
Merge pull request #557 from twisted/8855-rodrigc-ckeygen-py3

Author: rodrigc
Reviewer: einarfd
Fixes: #8855

Port ckeygen to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/557#pullrequestreview-4262846
